index.html

```diff
@@ -7,7 +7,35 @@
   </head>
   <body>
     <div class="gameBoy">
-      <div class="uiFrame"></div>
+      <div class="uiFrame">
+        <div class="gameplay">
+          <div class="petParamsBox">
+            <div class="petParams">
+              <img src="assets/heart.png" alt="heartIcon">
+              <p class="life counter">10</p>
+            </div>
+            <div class="petParams">
+              <img src="assets/eat.png" alt="heartIcon">
+              <p class="life counter">10</p>
+            </div>
+            <div class="petParams">
+              <img src="assets/sleep.png" alt="heartIcon">
+              <p class="life counter">10</p>
```

Comment:
**suggestion:** I would change the classnames to the proper life parameter names (I mean life, energy etc.). It will be easier for you later to catch them using JS

Reply:
Sure, obviously that was the plan, just forgot to do it and left "life" class everywhere.  I'll merge it that way and change it when adding js logic. ;)